### DISCUSSION TITLE
Speed Dial Close

### DISCUSSION THREAD

**Discussion by kytyvumyf** [darioielardi/flutter_speed_dial#106] [Q&A] [answered]
I want to close Speed dial after click on any action button.
Please tell me how can i do this i am stuck from many days.

**Comment by prateekmedia** [comment, marked as answer]
Use the open close dial property of the Speed dial, refer to readme for more info.

https://github.com/darioielardi/flutter_speed_dial#close-on-willpop

You can also use flutter hooks or other state management solutions to update the state of the speed dial if you want.
